### PR TITLE
Add card docs

### PR DESCRIPTION
**Pull request by paddyduke**
screencapture-localhost-9001-components-card-2018-06-15-10_02_58

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -141,7 +141,15 @@ module.exports = function(grunt) {
           'assets/scss/**/**/*.scss'
         ],
         tasks: ['sass']
-      }
+      },
```
Comment (paddyduke) [inline Gruntfile.js]: These changes relate to **[Get docs to reload on CSS change](https://github.com/fac/origin/pull/133/commits/16e06f74cc6bcc0c72a13b5d8a7312c3fe01061b)**

File: assets/scss/docs/_docs.scss [hunk 2]
```diff
@@ -88,15 +82,28 @@ h4 {
...
   font-size: 20px;
-  margin-top: 1em;
 }
 
 h6 {
   color: #5e6e78;
   font-size: 18px;
+}
+
+h2:not(:first-child),
```
Comment (paddyduke) [inline assets/scss/docs/_docs.scss]: More fixes for the docs slightly wonky heading and text spacing.

File: assets/scss/docs/_docs.scss [hunk 3]
```diff
@@ -346,16 +353,21 @@ figure {
 
 .DocsExample-preview,
 .DocsExample-render {
+  background-color: #fff;
```
Comment (paddyduke) [inline assets/scss/docs/_docs.scss]: Just removing some Origin mixins

File: assets/scss/docs/components/_docs-examples.scss [hunk 4]
```diff
@@ -19,159 +19,159 @@
  */
 
 .DocsExample-preview--amber {
-  @include background-color(amber);
+  background-color: #f5a300 !important;
```
Comment (paddyduke) [inline assets/scss/docs/components/_docs-examples.scss]: Removing mixins

File: docs/components/card.md [hunk 5]
```diff
@@ -0,0 +1,416 @@
...
+Cards are used to group related content and controls together to make pages easier to read and use.
+
+* Will be replaced with the ToC, excluding the "Contents" header
+{:toc}
+
+## Usage
+
+We use Cards both as large, page-scale containers for other content, and also for separating groups of smaller data blocks, such as on a dashboard view.
+
+By default, Cards fill up 100% of the available horizontal space (up to a max-width of `960px`), so you should use another layout element to create the layout you want, and drop the Cards into it.
```
Comment (craigcwaveformlabs) [inline docs/components/card.md]: Presuming we don't have a link to what this 'other layout element' is at the moment?

Comment (paddyduke) [reply docs/components/card.md]: Not yet. We still have to create it/them.

File: assets/scss/global/components/_CornerLayout.css [hunk 6]
```diff
@@ -0,0 +1,90 @@
+/**
+ * Use the `fe-CornerLayout` to lay out entity previews where there are three or
```
Comment (craigcwaveformlabs) [inline assets/scss/global/components/_CornerLayout.css]: Should this/will this have a document to describe it as a component, or pattern? Or is it more of a repeatable pattern that we'll pick up in usage.

File: docs/components/card.md (hunk 5 above)
Comment (craigcwaveformlabs) [inline docs/components/card.md]: Presuming we don't have a link to what this 'other layout element' is at the moment?

File: assets/scss/global/components/_CornerLayout.css (hunk 6 above)
Comment (paddyduke) [reply assets/scss/global/components/_CornerLayout.css]: I'm planning to add this in the next set of components/patterns, along with the `FlexLayout`, once we've shipped it with the Running Timers menu.

**Comment by craigcwaveformlabs** [comment]
Pass 👍 Looks good to me from the point of view of reading the documentation.

**Pull request merged**